faithcomesbyhearing/dbp-etl
# FCBHDBP-487

## Description
It has replaced the BiblePublisher code with sofria-client  code to get the bible verse text.

## NOTE
This PR depends on sofria-client PR:
- https://github.com/faithcomesbyhearing/sofria-cli/pull/9

We need that the sofria-client is on master because etl will clone sofria-cli using master branch.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-487

## How Do I QA This
You should run the following two examples, they return success and they should generate the following sql inserts related to the `bible_verses` entity:
1. **Turkmen, Southern_P1TUKTTV_USX**

```shell
python3 load/DBPLoadController.py test s3://etl-development-input "Turkmen, Southern_P1TUKTTV_USX"
```
- Outcome
[Trans-TUKTTVP_ET.sql.log](https://github.com/faithcomesbyhearing/dbp-etl/files/10060740/Trans-TUKTTVP_ET.sql.log)

2.  **Spanish_N2SPNTLA_USX**

```shell
python3 load/DBPLoadController.py test s3://etl-development-input Spanish_N2SPNTLA_USX
```

- Outcome
[Trans-SPNTLAN_ET.sql.log](https://github.com/faithcomesbyhearing/dbp-etl/files/10060759/Trans-SPNTLAN_ET.sql.log)
[Trans-SPNTLAO_ET.sql.log](https://github.com/faithcomesbyhearing/dbp-etl/files/10060760/Trans-SPNTLAO_ET.sql.log)
